docs/chymyst01.md

```diff
@@ -172,8 +173,11 @@ The new value of the counter (either `n+1` or `n-1`) will be carried by the new
 The previous counter molecule (with its old value `n`) will be consumed by the reactions.
 The `incr` and `decr` molecules will be likewise consumed.
 
+![Reaction diagram counter(n) + incr => counter(n+1) etc.](http://winitzki.github.io/joinrun-scala/counter-incr-decr.svg)
+
 It is important to note that the two reactions need to be defined together in a single call to `join`.
-The reason is that both reactions use the same input molecule `counter`.
+The reason is that both reactions contend on the same input molecule `counter`.
```

Comment:
Actually, I'm not very gifted in language. It took me 10 years of intensive self-study plus 5 years of graduate school to get my English to this point. And I still make mistakes.